src/deploy/functions/params.ts

```diff
@@ -584,6 +688,10 @@ async function promptResourceString(
 }
 
 type retryInput = { message: string };
+function shouldRetry(obj: any): obj is retryInput {
+  return (obj as retryInput).message !== undefined;
```

Comment:
Do you also need to verify that obj is indeed an object? It looks like the previous code this is replacing used to check for that.

Reply:
Good catch.